src/components/view/Contact/index.js

```diff
@@ -63,6 +65,7 @@ class Contact extends Component {
                         )}
                     </div>
                 )}
+                <CookieBanner />
```

Comment:
Should we put this up at the App level so it's available even if someone first visits with a deep linked URL?

Reply:
yea I'll change.